src/vocutouts/workers.py

```diff
@@ -116,16 +121,29 @@ def cutout_range(
     if any(math.isinf(v) for v in (ra_min, ra_max, dec_min, dec_max)):
         raise TaskFatalError("UsageError Unbounded ranges not yet supported")
     pos = SkyCoord(ra_min, dec_min, unit="deg")
-    xspan = (ra_max - ra_min) * 3600 / 0.2
-    yspan = (dec_max - dec_min) * 3600 / 0.2
+    xspan = ((ra_max - ra_min) * 3600 / 0.2) * u.dimensionless_unscaled
+    yspan = ((dec_max - dec_min) * 3600 / 0.2) * u.dimensionless_unscaled
     row = 1 * u.dimensionless_unscaled
     input_table = QTable(
-        [row, pos, xspan, yspan], names=["id", "position", "xspan", "yspan"]
+        [[row], [pos], [xspan], [yspan]],
+        names=["id", "position", "xspan", "yspan"],
     )
     butler.put(input_table, "cutout_positions", **data_id)
 
-    # Perform the cutout.
-    data_query = " AND ".join(f"{k}='{v}'" for k, v in data_id.items())
+    # Perform the cutout.  This currently uses subprocess to call pipetask.
+    # The long-term plan is for there to be an API for running a pipeline so
+    # that any setup overhead can be incurred once at worker start and then
+    # cutouts can be done with a direct Python call.
+    #
+    # This tells Butler to register data types every time, which as above
+    # should not be necessary, but they do have to be registered once.
+    data_query_terms = []
+    for key, value in data_id.items():
+        if isinstance(value, int):
```

Comment:
Isn't it safer to quote strings and leave non-strings unquoted?

The python API has bind parameters of course but that can't be used.

Reply:
It's a minor thing since once we have a Python API we can throw this out, but I think it's safer to quote as much as possible so that anything unexpected isn't interpreted as SQL.